src/dlr_relayvm.cc

```diff
@@ -87,14 +94,11 @@ void RelayVMModel::SetupVMModule(const std::vector<DLRModelElem>& model_elems) {
 
   tvm::runtime::PackedFunc init = vm_module_->GetFunction("init");
   if (ctx_.device_type == DLDeviceType::kDLCPU) {
-    init(static_cast<int>(ctx_.device_type), ctx_.device_id,
-         static_cast<int>(tvm::runtime::vm::AllocatorType::kPooled));
+    init(static_cast<int>(ctx_.device_type), ctx_.device_id, static_cast<int>(allocator_type_));
   } else {
     // CPU context also must be initialized because input/output data comes from CPU.
-    init(static_cast<int>(ctx_.device_type), ctx_.device_id,
-         static_cast<int>(tvm::runtime::vm::AllocatorType::kPooled),
-         static_cast<int>(DLDeviceType::kDLCPU), 0,
-         static_cast<int>(tvm::runtime::vm::AllocatorType::kPooled));
+    init(static_cast<int>(ctx_.device_type), ctx_.device_id, static_cast<int>(allocator_type_),
+         static_cast<int>(DLDeviceType::kDLCPU), 0, static_cast<int>(allocator_type_));
```

Comment:
Are there `GPU` (or other accel) specific allocators?

Reply:
Not yet, there are only two options - naive or pooled.